Doc/reference/datamodel.rst

```diff
@@ -2715,10 +2715,12 @@ left undefined.
    (``+=``, ``-=``, ``*=``, ``@=``, ``/=``, ``//=``, ``%=``, ``**=``, ``<<=``,
    ``>>=``, ``&=``, ``^=``, ``|=``).  These methods should attempt to do the
    operation in-place (modifying *self*) and return the result (which could be,
-   but does not have to be, *self*).  If a specific method is not defined, the
+   but does not have to be, *self*).  If a specific method is not defined, or if
+   that method returns NotImplemented, the
```

Comment:
```suggestion
   that method returns :const:`NotImplemented`, the
```

Properly format and cross-reference this, so readers can find more information and examples as desired.